src/instantsearch/instantsearch.ts

```diff
@@ -262,12 +264,12 @@ export class NgAisInstantSearch implements AfterViewInit, OnInit, OnDestroy {
     // custom algolia client agent
     if (!config.searchClient && !config.createAlgoliaClient) {
       const client = algoliasearch(config.appId, config.apiKey);
-      client.addAlgoliaAgent(`angular-instantsearch ${VERSION}`);
-
       config.searchClient = client;
       config.appId = undefined;
       config.apiKey = undefined;
     }
+    config.searchClient.addAlgoliaAgent(`angular (${AngularVersion.full})`);
```

Comment:
This will break with a custom `searchClient` that doesn't expose the `addAlgoliaAgent` function (we can add a specific test for that).